tutorials/Ocean/internal_wave.jl

```diff
@@ -98,14 +98,15 @@ time_step = 0.005 # close to Δx / c = 0.5 * 1/16, where Δx is nominal resoluti
 
 # and build a model with a smidgeon of viscosity and diffusion,
 
-using ClimateMachine.Ocean: HydrostaticBoussinesqSuperModel
+using ClimateMachine.Ocean.SuperModels: SplitExplicitOceanSuperModel
 
-model = HydrostaticBoussinesqSuperModel(
+model = SplitExplicitOceanSuperModel(
     domain = domain,
-    time_step = time_step,
-    initial_conditions = initial_conditions,
     parameters = NonDimensionalParameters(),
-    turbulence_closure = (νʰ = 1e-6, νᶻ = 1e-6, κʰ = 1e-6, κᶻ = 1e-6),
+    barotropic_time_step = time_step,
```

Comment:
Are we sure of these 2 time-step setting ? Would look better to me if it was switched, i.e,
 barotropic_time_step = time_step / 24
 baroclinic_time_step = time_step

Reply:
Agree, they got switched up! Even fixing that I'm having trouble finding stable values for these; but possibly I need to run this problem at higher resolution.